col/src/main/java/vct/col/ast/print/JavaPrinter.java

```diff
@@ -713,14 +714,19 @@ public void visit(DeclarationStatement s){
     s.getType().accept(this);
     out.printf(" %s", s.name());
     if (expr!=null){
-      out.printf("=");
+      out.printf(" = ");
       nextExpr();
       expr.accept(this);
     }
     if (!in_expr) out.printf(";");
   }
 
   public void visit(Method m){
+    if (m.kind == Method.Kind.Constructor && m.getBody() instanceof BlockStatement) { //empty constructors don't need to be printed
+      if (((BlockStatement) m.getBody()).isEmpty()) {
+        return;
+      }
+    }
```

Comment:
See <https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.8.9>, we should also check:
* the number of arguments is 0
* the method throws nothing
* there is no other constructor

Reply:
In method ```public void visit(Method m)``` I do not have information about other constructors. Do you want me to remove this if-statement, and just change back to the situation before this PR?

Reply:
I'm happy to assist in getting this to work better, but you can decide for yourself if it's worth it. With #541 we can probably also ensure that your formatting can happen before the default constructor is added instead.

Reply:
For now, I change it back to what it was before.